Exercise_Lab05/Exercise_Lab05/Lab5_1.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise_Lab05
+{   
+    /// <summary>
+    /// Bài 5.1 trong Lab05
+    /// Yêu cầu:
+    /// Nhập hiện mảng
+    /// Tìm giá trị nhỏ nhất trong mảng
+    /// Đảo các phần tử trong mảng
+    /// Sắp xếp mảng tăng, giảm
+    /// Tìm các số nguyên tố trong Mảng
+    /// Số lượng các sô dương liên tiếp nhiều nhất
+    /// Trung bình cộng các phần tử dương trong mảng
+    /// Mảng có các phần tử âm dương đan xen?
+    /// </summary>
+    class Lab5_1
+    {
+        public void ShowResult()
+        {
+            int[] arr = { 2, 45, -56, 32, 42, 342, -6, 7, 8, -2 };
+            Console.WriteLine("--------3.1----------");
+            Console.WriteLine("Mảng: ");
```

Comment:
int[] arr = { 2, 45, -56, 32, 42, 342, -6, 7, 8, -2 };
            Console.WriteLine("--------3.1----------");
            Console.WriteLine("Mảng: ");
            ShowArray(arr);

- chơi thủ công vậy e, thế nếu không fix cứng nữa thì làm như nào tức là cho người dùng nhập tay chứ không fix cứng 1 mảng như này nữa

Reply:
Em thêm hàm input này ạ!
        public int[] InputArray()
        {
            int[] arr = new int[10];
            for(int i = 0; i<10; i++)
            {
                bool check = false;
                Console.Write("a[{0}] = ", i);
                while (!check)
                {
                    try
                    {
                        arr[i] = Convert.ToInt32(Console.ReadLine());
                        check = true;
                    }
                    catch (FormatException)
                    {   
                        Console.WriteLine("Nhập sai vui lòng nhập lại: ");
                        check = false;
                    }
                }
                
            }
            return arr;
        }

Reply:
vẫn không thông minh lắm. cách này e vẫn giới hạn 10 phẩn tử ngta nhập nhiều hơn hoặc ít hơn thì thế nào

Reply:
đề bài nó bắt 10 phần tử mà a